sickchill/oldbeard/helpers.py

```
@@ -855,6 +855,56 @@ def decrypt(data, encryption_version=0):
+    if not value or value == "None":
+        return ""
+    if encryption_version is None:
+        encryption_version = settings.ENCRYPTION_VERSION
+    if encryption_version not in (1, 2):
+        return value
+    try:
+        decrypted = decrypt(value, encryption_version)
+        if encrypt(decrypted, encryption_version) == value.strip():
+            return decrypted
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)